app/components/filter_component.html.erb

```diff
@@ -60,6 +60,35 @@
                         </div>
                       </div>
                     <% end %>
+                     <% elsif filter[:type] == :checkbox_filter %>
```

Comment:
Maybe `searchable_checkbox`? 

Although is this strictly a different type of filter? It looks like this is a special case of the `checkbox` filter. Maybe we could pass in options that 1) mark it for the js searchbox enhancement and 2) render the filter tag links just above the filter.

So setting up the filter would be like:
```
{
  type: :checkboxes,
  searchable: true,
  display_filter_tags: true,
  heading: "Whatever",
  name: 'subjects',
  options: [blah],
}
```